src/Avalonia.Themes.Fluent/Controls/MenuItem.xaml

```diff
@@ -157,6 +159,7 @@
               </ContentPresenter.DataTemplates>
             </ContentPresenter>
             <Popup Name="PART_Popup"
+                   x:CompileBindings="False"
                    WindowManagerAddShadowHint="False"
                    MinWidth="{Binding Bounds.Width, RelativeSource={RelativeSource TemplatedParent}}"
```

Comment:
Using ReflectionBinding keyword instead of disabling compiled bindings should be easier to manage and search after. Also helps when there are two bindings or more.

Reply:
Sry messed the previous PR. Will fix it in the new one!